packages/@aws-cdk/aws-appmesh/lib/tls-certificate.ts

```diff
@@ -0,0 +1,199 @@
+import { ICertificate } from '@aws-cdk/aws-certificatemanager';
```

Comment:
No unqualified imports outside your module. 
```suggestion
import * as certificate from '@aws-cdk/aws-certificatemanager';
```
[Reference](https://github.com/aws/aws-cdk/pull/11563#discussion_r532950715)